apps/daffio/src/app/content/pwa/pages/pwa/pwa.component.html

```diff
@@ -0,0 +1,159 @@
+          <img daffFeatureIcon src="/assets/why-pwa/book-icon.svg" />
+          <p daffFeatureSubheader>Google Developers</p>
+          <h4 daffFeatureTitle>
+            PWA Case Studies
+          </h4>
+          <p daffFeatureSubtitle>
+            Learn why and how developers have used the web to create amazing web experiences for their users.
+          </p>
+        </daff-feature>
+      </a>
+      <a href="https://www.pwastats.com/" target="_blank" rel="noopener noreferrer">
+        <daff-feature class="understanding__feature">
+          <img daffFeatureIcon src="/assets/why-pwa/stat-icon.svg" />
+          <p daffFeatureSubheader>PWA Stats</p>
+          <h4 daffFeatureTitle>
+            PWA Conversion Stats
+          </h4>
+          <p daffFeatureSubtitle>
+            A community-driven list of stats and news focused on Progressive Web Apps.
+          </p>
+        </daff-feature>
+      </a>
+    </div>
+  </daff-container>
+</daff-callout>
+
+<daff-callout layout="centered">
+  <daff-container size="md">
+    <h3 daffCalloutTitle>
+      Who's building PWAs?
+    </h3>
+    <p daffCalloutSubtitle>
+      Brands are increasingly utilizing Progressive Web Apps to bridge the gap between native apps and websites to create better user experiences.
+    </p>
+    <div class="card-grid">
+      <a href="https://eng.uber.com/m-uber/" target="_blank" rel="noopener noreferrer">
+        <daff-card class="card-grid__card">
+          <img daffCardImage src="/assets/why-pwa/uber-logo.svg" />
+          <h4 daffCardTitle>Uber</h4>
+          <p>Uber built their PWA to minimize the client footprint while maintaining the rich experience of the native app.</p>
+        </daff-card>
+      </a>
+      <a href="https://app.starbucks.com/" target="_blank" rel="noopener noreferrer">
+        <daff-card class="card-grid__card">
+          <img daffCardImage src="/assets/why-pwa/starbucks-logo.svg" />
+          <h4 daffCardTitle>Starbucks</h4>
+          <p>The Starbucks PWA makes their service universally accessible to both established and emerging markets.</p>
+        </daff-card>
+      </a>
+      <a href="https://lancome-usa.com" target="_blank" rel="noopener noreferrer">
+        <daff-card class="card-grid__card">
+          <img daffCardImage src="/assets/why-pwa/lancome-logo.svg" />
+          <h4 daffCardTitle>Lanc&ocirc;me</h4>
+          <p>Lanc&ocirc;me's PWA aimed to drive traffic and re-engagement, which ultimately saw a 17% increase in conversions.</p>
+        </daff-card>
+      </a>
+    </div>
+  </daff-container>
+</daff-callout>
+
+<daff-callout layout="centered" class="comparison">
+  <daff-container size="md">
+    <h3 daffCalloutTitle>
+      The best solution for ecommerce
+    </h3>
+    <p daffCalloutSubtitle>
+      PWAs are a rapidly developing technology that is continuously seeing new progressive enhancements. It is on the fast-track to becoming the new standard for mobile-first solutions.
```

Comment:
I think "that sees continuous progressive enhancement" is easier to read. And I think "new" is unnecessary.

Reply:
Pretty minor, but I like "become" better in the last sentence: "It is on the fast-track to become the new standard for mobile-first solutions." Whichever you like better though